### PR TITLE
Benefits of contributing article - style review and minor updates 

### PR DESCRIPTION
**Pull request by Ludmila-N**

### CODE REVIEW

**Comment by rrrutledge** [comment]
Great!  Will look next week.

File: contributor/05-benefits-of-contribution-article.asciidoc [hunk 1]
```diff
@@ -14,72 +14,72 @@ their purview?
...
 to truly self select tasks to work on does play an important role.
 This self selection typically also leads to host projects being very welcoming
-and supportive as those are key to keeping motivation in contributors up.
+and supportive in their effort to attract motivated contributors.
 
 === Team level motivation
 
-Remember that annoying bug that's suddenly fixed upstream? Why should your
-team spend the extra effort to contribute that fix to the upstream project?
+Remember that annoying bug that's been suddenly fixed upstream? Why should your
```
Comment (lenucksi) [inline contributor/05-benefits-of-contribution-article.asciidoc]: Just from reading this now I wonder if we should add a word about the origin of this fix, i.e. that the host team actively created it and is maintaining it. Opinions here?

File: contributor/05-benefits-of-contribution-article.asciidoc [hunk 2]
```diff
@@ -14,72 +14,72 @@ their purview?
...
+long term dependencies which can only be influenced in a limited way). That way
+balancing reimplementing versus reuse becomes easier.
 
 === Company motivation
 
 Remember that functionality that's specific to your company domain - but that
 is maintained in mutiple implementations across the entire company? What if
-there were a way to avoid a dozen buggy implementations and merge them into a
-shared asset with a development process attached that avoids the usual
+there were a way to avoid a dozen buggy versions and merge them into a
```
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: It's a repeat word in two lines and reads like "buttery butter", but if you feel like it's needed for consistency, I will revert.

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 1 above)
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: I think "suddenly fixed upstream" does imply that "your team" had nothing to do with the fix other than enjoy the result, and I would not want to imply that it was done by the host team (what if there were other contributors?)
If I were to make one change here, I would change to "FINALLY fixed upstream" to indicate that "your team" was indeed suffering until that happened.
Thoughts?

File: contributor/05-benefits-of-contribution-article.asciidoc [hunk 3]
```diff
@@ -14,72 +14,72 @@ their purview?
...
 keeps working with your modifications and requirements.
 
 Having team members work as active contributors in projects your team depends on
-means that likely they will get to have a say in project direction and timelines,
+means that they may get to have a say in the project direction and timelines,
 which can be beneficial for your team.
 
 By using InnerSource teams can establish a middle path between "be independent
-and build your own" (including any number of new bugs that you own) and "save
+and build your own" (including any number of new bugs that you introduce) and "save
```
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: Reads clunky as "own" bears the stress in both sentences, so I tried to say the same with a different word. And yes, if you build your own and have a NEW bug (as the text already says), that means you have introduced it - I don't feel that mentioning it is somehow negative, given the context. 
I will revert back if that's important to keep.

File: contributor/05-benefits-of-contribution-article.asciidoc [hunk 4]
```diff
@@ -14,72 +14,72 @@ their purview?
...
 Aside from all of these objective criteria there is a component that is very
-hard to meassure but has been reported both in InnerSource and in Open Source
+hard to meassure, but has been reported both in InnerSource and in Open Source
 projects alike: people participate because they find work on those projects
-personally fullfilling and fun. Most likely, the aspect of being in a position
+personally fulfilling and fun. Most likely, the aspect of being in a position
 to truly self select tasks to work on does play an important role.
 This self selection typically also leads to host projects being very welcoming
-and supportive as those are key to keeping motivation in contributors up.
+and supportive in their effort to attract motivated contributors.
```
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: Changed to "and supportive in their effort to keep contributors motivated." - is this better?

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 1 above)
Comment (rrrutledge) [reply contributor/05-benefits-of-contribution-article.asciidoc]: SGTM

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (rrrutledge) [reply contributor/05-benefits-of-contribution-article.asciidoc]: I'm fine either way.

File: contributor/05-benefits-of-contribution-article.asciidoc [hunk 5]
```diff
@@ -14,72 +14,72 @@ their purview?
...
 Remember that functionality that's specific to your company domain - but that
 is maintained in mutiple implementations across the entire company? What if
-there were a way to avoid a dozen buggy implementations and merge them into a
-shared asset with a development process attached that avoids the usual
+there were a way to avoid a dozen buggy versions and merge them into a
+shared asset? What if the development process for this shared asset ran without the usual
 drain of energy that central dependencies bring to the table? Many open source
-projects are being used by a huge number of players - some of which participate
+projects are being used by a huge number of players - some of who participate
 in their design and development. Encouraging cross team collarboration in InnerSource
```
Comment (rrrutledge) [inline contributor/05-benefits-of-contribution-article.asciidoc]: Not a change that you introduced, but still want to suggest fixing this typo.

```suggestion
 in their design and development. Encouraging cross team collaboration in InnerSource
```

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: Not seeing anything from Johannes, I'm reverting it back to "own" for the sake of closing this PR.

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: Not seeing anything from Johannes, I'm reverting it back to "implementations" for the sake of closing this PR.

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 1 above)
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: Et fecit

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 5 above)
Comment (Ludmila-N) [reply contributor/05-benefits-of-contribution-article.asciidoc]: oops thank you!

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (lenucksi) [reply contributor/05-benefits-of-contribution-article.asciidoc]: > Changed to "and supportive in their effort to keep contributors motivated." - is this better?

Yes, that change is better. Thanks!

File: contributor/05-benefits-of-contribution-article.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (lenucksi) [reply contributor/05-benefits-of-contribution-article.asciidoc]: I'm fine with regular non-extra buttery butter then too .

**Pull request merged**